backend/dal/Migrations/Initial/PostDeploy/01-Tasks.sql

```diff
@@ -114,6 +114,38 @@ INSERT INTO dbo.[Tasks] (
     , 13
     , 0
     , 2
+), (
+    14
+    , 'Appraisal ordered'
```

Comment:
Since these are essentially duplicates of the original assessment workflow, are you copying the values over?  I don't think it's necessary for this iteration, but curious.

Reply:
Yes, the frontend copies them over when approved for SPL -> Pre-Marketing